src/tiles/item-image.ts

```diff
@@ -26,13 +26,20 @@ export class ItemImage extends LitElement {
   @query('img') private baseImage!: HTMLImageElement;
 
   render() {
+    const width = () => {
+      if (this.isListTile && !this.isCompactTile) return '100px';
```

Comment:
blocker: the `.mobile` class changes the size of the `<div id="thumb">`
100px -> 90px
45px -> 30px

Reply:
if `this.isListTile` the width of the waveform is always going to be `100%`

Reply:
fixed it